recipes/wstools/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "wstools" %}
+{% set version = "0.4.5" %}
+{% set sha256 = "499760d87d9bb62fb937669bb9acb2de8a6393dcce3c5fbd3bc693b4a7bdd4ba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It looks like `0.4.5` imports `pip` in its `setup.py`. In the latest code on github that requirement goes away.